run/test/specs/utils/before_test_setup.ts

```diff
@@ -1,15 +1,12 @@
+import { W3CCapabilities } from '@wdio/types/build/Capabilities';
 import XCUITestDriver, { XCUITestDriverOpts } from 'appium-xcuitest-driver/build/lib/driver';
 import { DeviceWrapper } from '../../../types/DeviceWrapper';
-import { User } from '../../../types/testing';
-import { openAppThreeDevices } from './open_app';
 import { runScriptAndLog } from './utilities';
-import { CapabilitiesIndexType } from './capabilities_ios';
-import { W3CCapabilities } from '@wdio/types/build/Capabilities';
 
-export const beforeTestSetupGroup = async (device: DeviceWrapper) => {
-  // const { device1, device2, device3 } = await openAppThreeDevices(platform);
-  // return { device1, device2, device3 };
-};
+// export const beforeTestSetupGroup = async (device: DeviceWrapper) => {
+//   // const { device1, device2, device3 } = await openAppThreeDevices(platform);
+//   // return { device1, device2, device3 };
+// };
```

Comment:
can be removed?

Reply:
removed and renamed file to just be permissions